### PR TITLE
Update for latest version of widget-core

### PR DESCRIPTION
**Pull request by agubler**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [ ] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [x] Unit or Functional tests are included in the PR

**Description:**

Update for latest widget-core

Depends on release of widget-core

### CODE REVIEW

**Review by dylans — APPROVED** [review]
Looks reasonable to me.

**Review by kitsonk — APPROVED** [review]
:+1:

File: src/support/d.ts [hunk 1]
```diff
@@ -15,6 +19,9 @@ export function assignProperties(target: HNode, properties: VirtualDomProperties
 export function assignProperties(target: WNode, properties: WidgetProperties): WNode;
 export function assignProperties(target: WNode | HNode, properties: WidgetProperties | VirtualDomProperties): WNode | HNode;
 export function assignProperties(target: WNode | HNode, properties: WidgetProperties | VirtualDomProperties): WNode | HNode {
+	if (isVirtualDomPropertiesWithClasses(properties) && typeof properties.classes === 'function') {
+		assign(properties, { classes: properties.classes() });
```
Comment (kitsonk) [inline src/support/d.ts:23]: We don't seem to be hitting this code.

Comment (agubler) [reply src/support/d.ts:23]: This should be covered now.

**Review by kitsonk — APPROVED** [review]

**Pull request merged**